lib/indacoin.ex

```diff
@@ -305,13 +305,25 @@ defmodule Indacoin do
     - _tx_id_ :: integer
   """
   def valid_callback_signature?(indacoin_signature, indacoin_nonce, user_id, tx_id) do
-    new_signature =
-      "#{partner_name()}_#{user_id}_#{indacoin_nonce}_#{tx_id}"
-      |> sign()
-      |> Base.encode64()
-      |> to_string()
+    callback_signature(indacoin_nonce, user_id, tx_id) == to_string(indacoin_signature)
+  end
+
+  @doc """
+  Create a callback signature from Indacoin.
 
-    new_signature == to_string(indacoin_signature)
+  ## params
+
+  Required request params:
```

Comment:
The params here are not request's params, therefore I would suggest omitting the `request` word here.